app/views.py

```diff
@@ -51,8 +64,23 @@ def page_server_error(error):
 
 
 def convert_to_readable_time(time):
-    return time.strftime('%a %b %d,%Y %I:%M:%S %p')
-
-
-def display(forecast):
-    return 'Time: {}\nTemperature: {}°C\nHumidity: {}\nProbability of Precipitation: {}\nWeather Summary: {}'.format(convert_to_readable_time(forecast.time), round(forecast.temperature, 2), forecast.humidity, forecast.precipProbability, forecast.summary)
+    return time.strftime('%b/%d/%Y %a %I:%M:%S %p')
+
+
+def display(forecast, time_method):
+    if time_method == 'currently':
+        return 'Time: {}\nTemperature: {}°C\nHumidity: {}\nProbability of Precipitation: {}\nWeather Summary: {}\
+        '.format(convert_to_readable_time(forecast.time), round(forecast.temperature, 2), forecast.humidity, forecast.precipProbability, forecast.summary)
+
+    info = ''
+    count = 0
+    for item in forecast.data:
+        info += '\n  time: {}'.format(convert_to_readable_time(item.time))
+        info += '\n  temperature: {}'.format(item.temperature)
+        info += '\n  humidity: {}'.format(item.humidity)
+        info += '\n  precipProbability: {}'.format(item.precipProbability)
+        info += '\n  summary: {}\n'.format(item.summary)
```

Comment:
@mknecht How is it used for concatenation? I can't find a good example for concatenation with triple double quote... <code>+=</code> is most suggested